src/chrome/lib/destroy.js

```diff
@@ -1,15 +1,14 @@
 // Script injected into the page to trigger removal of any existing instances
 // of the Hypothesis client.
-(function () {
```

Comment:
This script is now run in an isolated world as a content script. That's why it destroys the annotator by sending a DOM event rather than accessing `window.annotator` directly - which it can't see.